docs/publishers/cli.md

```diff
@@ -0,0 +1,65 @@
+# DPM: the datapackage manger CLI
+
+The dpm is a command-line tool aimed to help publishers to prepare and upload data to the Datapackage Registry. With dpm you will be able to:
+
+* Validate your data to ensure its quality
+* Upload datapackage
+* Tag uploaded datapackage to create historical snapshot
+* Remove uploaded datapackage that is no longer needed
+
+## Installation
+
+Dpm is available on pypi [hopefully soon it will be], you can install it using pip in the terminal:
+```
+[sudo] pip install dpm
+```
+
+**NOTE**
+
+You can alternatively install unstable version directly from the code repository:
+
+```
+[sudo] pip install git+https://github.com/frictionlessdata/dpm-py.git
+```
+
+## Configuration
+
+Dpmpy can be configured using `dpmpy configure` command. It will ask you
+to provide username, access_token and server address of datapackage registry.
+
+The config is stored in `~/.dpm/config`, you can edit it with text editor.
+Simple example config file can look like this:
+
+```
+username = myname
+access_token = mykey
+server = https://staging.datapackaged.com
+```
+
+## Usage
+
+### Publish
+
+To publish datapackage, go to the datapackage directory (with `datapackage.json`) and
+launch `dpmpy publish`. If your configured *username* and *access_token* are correct, dpmpy will
+upload datapackage.json and all relevant resources to the registry server.
+
+[TODO] add details in the [upload](upload/) section
+
+### Tag
+
+To create historical snapshot of your data, you can tag previously uploaded datapackage on the server. Use `dpmpy tag` command:
+```
+$ cd datapackage-dir
+$ dpmpy tag v1.1
+```
+This will copy the latest version of the datapackage to the separate location on the bitstore, so that this data will not be overwritten the next time you publish.
```

Comment:
BitStore not bitstore.

Also tweak meaning a bit here:

This will copy the latest version of the Data Package to a separate location in the BitStore. This  way you will be able keep a copy of your Data Package at this particular point in time.